### ISSUE TITLE
Error build apk/aab

### ISSUE THREAD

**Issue by Frabassa** [flet-dev/flet#3372]
I tried to build apk or aab but the console give the following error:
compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.

I have Android Studio in windows 11, and Flutter 3.22.1

**Comment by J-PRS** [comment]
Are you sure this is an error and not just a warning?

**Comment by FeodorFitsner** [comment]
It is fixed in Flet 0.23.1.

Please cleanup Cookiecutter cache before running `flet build` command.

On Windows:

```
rd /s /q %USERPROFILE%\.cookiecutters
```

On Mac/Linux:

```
rm -rf ~/.cookiecutters
```
